### PR TITLE
Make data in serialization benchmark static

### PR DESCRIPTION
**Pull request by sylwiaszunejko**
This PR addresses some problem that where missed during the review process of the initial PR adding serialization benchmark #295 
Fixed issues:
* All data should be static, or generated in such a way that every ran your will get same data
* Marshal and unmarshal tests should be done on the same type, fixed for duration

### CODE REVIEW

File: testdata/serialization/medium_json_data.json [hunk 1]
```diff
@@ -0,0 +1 @@
...F","Ukahc":"QxzeTkOnIv","VGTvu":"ONHCuyZlUZ","VeTPw":"SunozvXABk","VlXDc":"cZdRdkufsM","VrctV":"tlJyEICvIY","WFXgr":"vkeOsIfmZr","WejDD":"HXXDAqRxzB","YBNZI":"wamwcCOPcw","ZBUHl":"BXrBQcJXfs","ZNFnW":"DRkColJJLv","ZYzdd":"qYNrMnHudn","ZbfmL":"hMcyhhKRoB","dWTGB":"oereJzBFZN","dmhtV":"cdyXdgXLrN","dyDUh":"HqjfCLkkQD","eAlVa":"VSLpIBFXkx","ejbXs":"adnaZVZSfM","eznlM":"SkjcejRpzf","guIuQ":"nVISjNXnFX","hCtto":"uXyQBVlTrb","hDyYk":"pqHCvUgure","hGElt":"fVkIcUIeVZ","hdgry":"aKBomWYWvL","hnrZE":"cUheaAgBiw","jKQZu":"kijxuhgLTG","jyAeD":"AGeHBIWfzW","kAINt":"vVovUkUWVn","kGVaC":"GgRphTCXPo","kyrZk":"pVAJWlRfBN","lPTfL":"uNZuGfPlQC","mzbFt":"MVGqkiawiz","nGfMH":"yuBSyxThoB","naLLI":"sCyrafZvLb","oKSzI":"wpHtJyASyK","pGGUf":"QWPqNVBGVF","pJuQQ":"ekblLOHrWn","pLAhe":"TucCjMeGVE","psiWn":"tvVfzXJdRb","pvMdX":"AARLxizqdU","qXcYp":"SEMFlhXZDr","qsGAR":"AhSzpKTKvw","rzPWa":"uTALQrsetT","slRwW":"whNSaHmpvG","swrYw":"ROpIqXOuFW","tobIP":"fyYEYpldkU","uirHl":"VKGqGmvvSc","uqBEz":"gTblROZoji","usuPs":"cnjqzCFmGd","vHgDz":"PsGzazdxiX","wqNDB":"ZCzkgBVRnW","xPnvb":"lYnVPCYWOz","xQwZA":"UKGiutEReo","xRxjp":"LBZxpcePok","xrGDk":"hKFHCULlBu","yAZiz":"wmwYhRBlKA","ylxSm":"BEEVhuLKxf","zCgtm":"bNBmOcshos"}
```
Comment (dkropachev) [inline testdata/serialization/medium_json_data.json:1]: Let's move these files closer to the test, to the `tests/bench/testdata/*`

File: tests/bench/bench_marshal_test.go [hunk 2]
```diff
@@ -1,36 +1,55 @@
...
+		Name:     "Small",
+		FilePath: "../../testdata/serialization/small_binary_data.bin",
+	},
+	{
+		Name:     "Medium",
+		FilePath: "../../testdata/serialization/medium_binary_data.bin",
+	},
+	{
+		Name:     "Big",
+		FilePath: "../../testdata/serialization/big_binary_data.bin",
```
Comment (dkropachev) [inline tests/bench/bench_marshal_test.go:28]: it is not ok to save big files in git without good reason.
Let's generate json data by [gofakeit](https://github.com/brianvoe/gofakeit/): here is [example](https://github.com/brianvoe/gofakeit/blob/master/json_test.go)
Don't roget to set `Seed` to make it generate same data all the times.

For binary use following construction:
```
	rnd := rand.New(rand.NewSource(100))
	randomBuffer := make([]byte, size)
	_, err := io.ReadAtLeast(rnd, randomBuffer, size)
```

**Review by dkropachev — CHANGES_REQUESTED** [review]

File: tests/bench/bench_marshal_test.go (hunk 2 above)
Comment (sylwiaszunejko) [reply tests/bench/bench_marshal_test.go:28]: should I have smaller exampes saved in files and larger ones generated using the tool that you mentioned or should I sue these tools for every size data?

Comment (dkropachev) [reply tests/bench/bench_marshal_test.go:28]: No need to, if we have logic to generate data, why not to generate all of it.

**Review by dkropachev — APPROVED** [review]

**Pull request merged**